open-metadata-implementation/governance-servers/security-sync-services/security-sync-services-api/src/main/java/org/odpi/openmetadata/securitysyncservices/model/Ranger/RangerAttributes.java

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
Should not hav e Ranger specific conent in main Egeria code path - this should be in a connector.